src/main/webapp/js/components/dev/ExperimentsTable/.bower.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
@jrmartin why is there this file (.bower.json) as well as bower.json?  Are both necessary?